.github/workflows/generate.yaml

```diff
@@ -0,0 +1,52 @@
+name: Generate and Test on OAS change
+
+on:
+  pull_request:
+    paths:
+      - 'openapi.json'
+      - 'api.yaml'
```

Comment:
this shouldn't be hard-coded. It should be based on `konfig.yaml#specInputPath / specPath`

Reply:
what is the difference between specPath and specInputPath?

Reply:
[`specInputPath` ](https://github.com/konfig-dev/konfig/blob/main/generator/konfig-dash/packages/konfig-lib/src/KonfigYaml.ts#L482-L484)

Reply:
gotcha - so I want specInputPath if present, otherwise specPath

Reply:
yup. Everything should be parameterized by `konfig.yaml` avoid hard-coded filenames/paths everywhere

Reply:
definitely. makes sense